### DISCUSSION TITLE
i see myself invisible.. please help!

### DISCUSSION THREAD

**Discussion by petITA1185** [Draylar/identity#547] [Q&A]
i see myself invisible, morphed and not morphed. i have version 2.6.0 and i tried downloading 2.6.1        what does make me invisible? another mod..?        video: https://drive.google.com/file/d/1hHtYwuxmWCD4gzLj1Smm8odluBCn4k-x/view?usp=drive_link

**Comment by petITA1185** [comment]
if i remove the mod and put it again i turn visible again, but everytime i open the menu of morhping it makes me invisible. not even the time for me to morph, that im invisible.

**Reply by chigle-droid** [reply]
a similar thing is happening to me. when I open the menu, I turn invisible though this only happens on LAN or multiplayer, but this is a problem because I'm playing on a server that uses this mod.
